extensions/amp-consent/amp-consent.md

```diff
@@ -78,7 +78,11 @@ Example:
   "consents": {
     "my-consent": {
       "checkConsentHref": "https://example.com/api/show-consent",
-      "promptUI": "consent-ui"
+      "promptUI": "consent-ui",
+      "timeout": {
```

Comment:
Timeout doesn't go here. It goes to "policy"

Reply:
:-( policy is not yet mentioned anywhere, could you add one?

Reply:
wait, why timeout is at policy level?
timeout is directly linked to consent prompt ui, which is a consent instance level thing